utils/rx-page-objects/src/rxCopy.exercise.js

```
@@ -0,0 +1,157 @@
+var _ = require('lodash');
+
+/**
+ * @description rxCopy exercises.
+ * @see rxCopy
+ * @exports exercise/rxCopy
+ * @param {Object} options - Test options. Used to build valid tests.
+ * @param {rxSelect} options.instance - Component to exercise.
+ * @param {Boolean} [options.isPresent=true] - Determines if the component is present.
+ * @param {Boolean} [options.isDisabled=false] - Determines if the component is disabled.
+ * @param {Boolean} [options.isDisplayed=true] - Determines if the component is displayed.
+ * @param {ElementFinder} [testCopyArea=] - An input element somewhere that rxCopy contents can be pasted to.
+ * @param {String|Regex} [expectedText=] - The expected text content inside the input element to be copied.
+ * Can be a regular expression if the text content is very large.
+ */
+exports.rxCopy = function (options) {
+    if (options === undefined) {
+        options = {};
+    }
+
+    options = _.defaults(options, {
+        isPresent: true,
+        isEnabled: true,
+        isDisplayed: true
+    });
+
+    return function () {
+        var component;
+
+        /**
+         * @returns {Promise<String>}
+         */
+        function getPastedValue () {
+            return options.testCopyArea
+                .clear()
+                .sendKeys(protractor.Key.chord(
+                    (process.platform === 'darwin' ? protractor.Key.META : protractor.Key.CONTROL),
+                    'v'
+                ))
+                .getAttribute('value');
+        }
+
+        before(function () {
+            component = options.instance;
```

Should we throw an error if an instance is not provided? Or is this `exercise` only meant to be consumed in these files?

I imagine a default value would be hard to come up with